test/duplicates-v3.e2e-spec.ts

```diff
@@ -100,6 +100,9 @@ describe('Deposits in case of duplicates', () => {
       .mockImplementation(() => Promise.resolve());
 
     // deposit cache mocks
+    jest
```

Comment:
What was changed in the PR to make this mock necessary?

Reply:
before commit 571dc751094e2f7f0734c1204203d2307f084712 tests worked

Reply:
because of the index check before inserting the event to the deposit tree